docs/medhelm.md

```diff
@@ -348,7 +326,7 @@ MedHELM allows you to define and run custom LLM benchmarks by combining a few si
 > **Note:** This feature requires **crfm-helm >= 0.5.8**.  
 > Make sure you upgrade before continuing:  
 ```bash
-pip install --upgrade crfm-helm
+pip install -U crfm-helm
```

Comment:
Generally, I prefer using full flags over shortcut flags in documentation because they are clearer for the reader.